cmake/PythonModules.cmake

```diff
@@ -34,10 +34,8 @@ macro(py_exec)
     endif()
 endmacro()
 
-set(PYBIND11_NOPYTHON On)
-# this wont work if pybind11 is git submodule
-#find_package(pybind11 REQUIRED)
-
+# NOTE: this property must be set before including pybind11
```

Comment:
This comment seems confusing given the commented line below. Please check if it's okay and explain in the comment or fix.